flake.nix

```diff
@@ -26,6 +26,13 @@
 
     unionmount.url = "github:srid/unionmount";
     unionmount.flake = false;
+
+    emanote-template.url = "github:srid/emanote-template";
+    # Don't care about the output of emanote-template, only the source
+    emanote-template.flake = false;
+    # To filter emanote-template source
+    # Why nix-filter? Try filtering directories with `builtins.path`, then you will see why.
+    nix-filter.url = "github:numtide/nix-filter";
```

Comment:
I need to take a look at this, but briefly: what about https://www.tweag.io/blog/2023-11-28-file-sets/ ?

Reply:
`nix-filter` looked simpler to me on the first glance, I will have a look at `file-sets` again.